modules/logging-release-notes-5-9-5.adoc

```diff
@@ -0,0 +1,18 @@
+//module included in logging-5-9-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-9-5_{context}"]
+= Logging 5.9.5
+This release includes link:https://access.redhat.com/errata/RHBA-2024:5122[OpenShift Logging Bug Fix Release 5.9.5]
+
+[id="logging-release-notes-5-9-5-bug-fixes"]
```

Comment:
If you want to link to bug fixes anywhere, you can add a content tag here.

Reply:
@ShaunaDiaz I will discuss this with the team if that's required. We might start doing it with the next set of z-streams. :)